lib/daru/io/io.rb

```diff
@@ -275,6 +275,17 @@ def html_table_to_dataframe(table)
           order: table[:order],
           name: table[:name]
       end
+
+      CSV::Converters[:boolean_converter] = lambda do |f, _|
```

Comment:
Well, despite the fact that docs explicitly say "This Hash is intentionally left unfrozen and users should feel free to add values to it that can be accessed by all CSV objects", I don't believe any "global" solution is good or even necessary. 

Imagine large project that uses daru AND some other code to work with CSV. What's the probability they also have defined `:boolean_converter` with slightly different behavior (for ex., converting `'yes'` and `'no'` too)?..

Let's just have a constant with lambda in it?